mne/datasets/sleep_physionet/temazepam.py

```diff
@@ -0,0 +1,112 @@
+# Authors: Alexandre Gramfort <alexandre.gramfort@inria.fr>
+#          Joan Massich <mailsik@gmail.com>
+#
+# License: BSD Style.
+
+import numpy as np
+
+from ...utils import verbose
+from ._utils import _fetch_one, _data_path, BASE_URL, TEMAZEPAM_SLEEP_RECORDS
+from ._utils import _check_subjects
+
+SLEEP_RECORDS = 'physionet_sleep_records.npy'
```

Comment:
this should disappear

Reply:
+1 i thought i had remove them all. my bad.